apps/api/src/routers/actions_cache.py

```
@@ -11,7 +11,7 @@
 
 
 @router.post("/{owner}/{repo}/actions-caches", response_model=CacheListResponse)
-def list_caches(owner: str, repo: str, payload: CacheListInput, _user: UserOut = Depends(require_auth)):
+def list_caches(owner: str, repo: str, payload: CacheListInput, _user: UserOut = Depends(require_owner)):
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)